cmake/cmakepp_core/utilities/call_fxn.cmake

```diff
@@ -15,10 +15,29 @@ include(cmakepp_core/utilities/unique_id)
 # :param *args: The arguments to forward to the function.
 #]]
 function(_cpp_call_fxn_guts _cfg_fxn2call _cfg_result)
+    # Create a new arg list that is a copy of ARGN except all args of type desc
```

Comment:
I don't see why you can't double quote all of the arguments. CMake just uses double quotes for grouping so it shouldn't matter what you double quote. This leads to the much simpler:
```cmake
set(_cfg_args_list "")

foreach(_cfg_current_arg ${ARGN}) # Loop over all args
    string(APPEND _cfg_args_list " \"${_cfg_current_arg}\" ")
endforeach()
```

Reply:
I fixed this :+1: